src/sensai/evaluation.py

```diff
@@ -344,12 +344,11 @@ def _createResultData(self, trainedModels, evalDataList, testIndicesList, predic
 
 def computeEvaluationMetricsDict(model, evaluatorOrValidator: Union[VectorModelEvaluator, VectorModelCrossValidator]) -> Dict[str, float]:
     if isinstance(evaluatorOrValidator, VectorModelEvaluator):
-        evaluator: VectorModelEvaluator = evaluatorOrValidator
```

Comment:
Please avoid purely cosmetic changes like this (whether it's an improvement is debatable). We don't need to have changesets that complicate the history for things that aren't really an improvement.

Reply:
It's gone